test/unit/lib/plugins/aws/deployFunction.test.js

```diff
@@ -232,16 +233,24 @@ describe('AwsDeployFunction', () => {
         },
         layers: ['arn:aws:lambda:us-east-1:123456789012:layer:layer:1'],
       };
-
+      awsDeployFunction.serverless._logDeprecation = sinon.spy();
       awsDeployFunction.options = options;
 
       await awsDeployFunction.updateFunctionConfiguration();
 
       expect(normalizeArnRoleStub.calledOnce).to.be.equal(true);
       expect(normalizeArnRoleStub.calledWithExactly('arn:aws:iam::123456789012:role/Admin'));
       expect(updateFunctionConfigurationStub.calledOnce).to.be.equal(true);
-      expect(
-        updateFunctionConfigurationStub.calledWithExactly('Lambda', 'updateFunctionConfiguration', {
+      sinon.assert.calledWith(
```

Comment:
I think we could skip these assertions on deprecations - we usually don't do them and it's also expanding a test that is written in "old-way"

Reply:
I see that we still have that test expanded, is there a reason for that?